core/csv_api.php

```diff
@@ -438,3 +438,35 @@ function csv_format_due_date( BugData $p_bug ) {
 function csv_format_sponsorship_total( BugData $p_bug ) {
 	return csv_escape_string( $p_bug->sponsorship_total );
 }
+
+/**
+ * return the attachment count for an issue
+ * @param BugData $p_bug A BugData object.
+ * @return string
+ * @access public
+ */
+function csv_format_attachment_count( BugData $p_bug ) {
+	# Check for attachments
+	$t_attachment_count = 0;
+	if( file_can_view_bug_attachments( $p_bug->id, null ) ) {
```

Comment:
I wonder if we should move these access checks inside the API that returns the count, rather than having to check before it is called.  I can also imagine a scenario where a user can view their own attachments but not other people's (though this feature is not implemented yet), in such case having the API take the access check call into consideration makes sense.